dataland-frontend/src/components/resources/frameworkDataSearch/euTaxonomy/EuTaxonomyForNonFinancialsDataModel.ts

```diff
@@ -0,0 +1,642 @@
+import { type Category } from "@/utils/GenericFrameworkTypes";
+import { DropdownDatasetIdentifier, getDataset } from "@/utils/PremadeDropdownDatasets";
+
+export const euTaxonomyForNonFinancialsDataModel = [
+  {
+    name: "general",
```

Comment:
In the display page this is called "Basic Information"

Reply:
I think in the display page data model file the naming was manually altered as it was implemented in the View Page story. We have not changed the naming here, but use the naming from the dev tools.

Reply:
But this leads to the upload page using different naming than the view page. That is confusing at best.

Reply:
I agree. But if we change it here, i think we also have to change the backend name of the category, because if i change it here to basicInformation the backend won't be able to receive the field. If we change the backend names then this also would mean to migrate the existing data to the new field names.

Reply:
as discussed, no changes to the backend data model. Just make the frontend align with it and be consistent between upload and view page

Reply:
basic information is now general for all view page related stuff